code-challenges.js

```diff
@@ -20,49 +21,105 @@ const fruit2 = "banana"
+// let padresCombinedArrays 
+// //combine both with .concat and then add .length to the end
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4];
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3];
+// padresCombinedArrays = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns).length;
+// //var homeRuns = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+// console.log(padresCombinedArrays)
 
-// Describe your process:
 
-const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
-const givenValue1 = 42
-// Expected output: 7
+// //I cannot figure out number 2 for the life of meeeee
+// //okay it was a dumb mistake, I forgot to add: let padresCombinedArrays...
+// // Expected output: 9, actual outcome = 9 
 
-const givenValue2 = 10
-// Expected output: 8
 
-// --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
-// Describe your process:
 
-const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
-// Expected output: [82, 80, 79, 77, 76, 73, 72]
 
-const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
-// Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+// // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+
+// // Describe your process:
+
+// const currentCohort = ["Foxtrot 2022"]
+// // first you want to split all of the elements using .split
+// let split = "Foxtrot 2022".split("")
+// console.log(split)
+// // outcome = "F", "o", "x", "t", "r", "o", "t", "2", "0", "2", "2"
+// // then reverse the elements of the split array
+// let reversedArray = split.reverse()
+// console.log(reversedArray)
+// // outcome = 2 2 0 2 t o r t x o f in "" separated by commas 
+// //then you'd want to join the reversed and split elements using .join
+// let joinedElements = reversedArray.join("")
+// console.log(joinedElements)
+
+// // Expected output: "2202 tortxoF"
+// actual outcome was as expected
+
+
+
+// // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
+
+// // Describe your process: you'd want to use .lastIndexOf in order to get the outcome of 7
+
+// const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+// const givenValue1 = 42
+// console.log(myNumbers.lastIndexOf(givenValue1))
+
+// // // Expected output: 7, actual outcome = 7
+
+// // you would do the same thing for the next portion of this question, except you would want to type (givenValue2) instead
+// const givenValue2 = 10
+// console.log(myNumbers.lastIndexOf(givenValue2))
+// // Expected output: 8, actual outcome = 8
+
+// // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
+
+// // Describe your process: sort the numbers from biggest to smallest using .sort()
```

Comment:
Once again, this is the bare minimum for pseudocode and in future weeks will not be accepted.